package.json

```diff
@@ -15,7 +15,8 @@
     "lint": "eslint . --cache --fix --ext .ts,.tsx,.js",
     "format": "prettier --loglevel warn --write \"**/*.{ts,tsx,css,md}\"",
     "prepare": "husky install",
-    "postversion": "node ./scripts/manifest-version.js"
+    "postversion": "node ./scripts/manifest-version.js",
+    "update-dev-manifest": "node ./scripts/update-dev-manifest.js"
```

Comment:
i think we can avoid putting this in `package.json`. We can call the script directly from the workflow, right?